github-actions/rebase-pr/entrypoint.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+set -e
+
+# Set email and name config
+git config --global user.email "devops+rebase@angular.io"
+git config --global user.name "Angular Rebase Bot"
+
+# Exit for code a successful run
+SUCCESS_EXIT_CODE=0
+
+# The repository name and branch name for the PRs head and base
+HEAD_REPOSITORY=$(jq -r ".pull_request.head.repo.full_name" "$GITHUB_EVENT_PATH")
+HEAD_BRANCH=$(jq -r ".pull_request.head.ref" "$GITHUB_EVENT_PATH")
+BASE_REPOSITORY=$(jq -r ".pull_request.base.repo.full_name" "$GITHUB_EVENT_PATH")
+BASE_BRANCH=$(jq -r ".pull_request.base.ref" "$GITHUB_EVENT_PATH")
+
+# Change to the github workspace and initialize the repo if needed.
+cd "$GITHUB_WORKSPACE"
+git init
+
+# Add head and base repos as remotes, then fetch the remote branches
+git remote add head https://x-access-token:$TOKEN@github.com/$HEAD_REPOSITORY
```

Comment:
`HEAD` already has a meaning in git (and in case-insensitive systems, such as Windows, it is the same as `head`). Consider using a different name for the remote.